Makefile

```diff
@@ -66,13 +72,7 @@ install-extras:
 #=> test: execute tests
 .PHONY: test
 test:
-	python setup.py pytest
-
-#=> tox: execute tests via tox
-.PHONY: tox
-tox:
-	tox
-
+	python -m pytest tests
```

Comment:
I can't comment on the exact line, but this
https://github.com/biocommons/anyvar/blob/ec9fbf1c794bb138edae1f456e5621dd0ca13d7b/Makefile#L47

should be changed to 

```
pip install -e .[dev,test]
```

Or we can change setup.cfg to combine both `dev` and `test` into `dev`